argilla/src/argilla/_helpers/_datasets_server.py

```diff
@@ -0,0 +1,175 @@
+import requests
+import warnings
+from enum import Enum
+from typing import Any, Dict, Optional, TYPE_CHECKING, Union, List
+
+from argilla._exceptions._hub import DatasetsServerException
+
+if TYPE_CHECKING:
+    from argilla import Settings
+
+DATASETS_SERVER_BASE_URL = "https://datasets-server.huggingface.co"
+DATASETS_SERVER_HEADERS = {"Accept": "application/json"}
+DATASETS_SERVER_TIMEOUT = 30
+
+
+class FeatureType(Enum):
+    """Feature types supported by Argilla in the datasets server."""
+
+    TEXT = "text"
+    CHAT = "chat"
+    IMAGE = "image"
+    LABEL = "label"
+    INT = "int"
+    FLOAT = "float"
+
+
+def _get_dataset_features(repo_id: str, config: Optional[str] = None) -> Dict[str, Dict[str, Any]]:
```

Comment:
Regarding this comment https://github.com/argilla-io/argilla/pull/5472/files#r1751978426
```suggestion
def _get_dataset_features(repo_id: str) -> Dict[str, Dict[str, Any]]:
```